scripts/constant/abis.ts

```diff
@@ -1,4 +1,4 @@
-module.exports = {
+export const abis: Record<string, any> = {
```

Comment:
@pradyuman-verma Maybe let's export it by default?

Reply:
Because we changed for addresses, right? Otherwise, we can change addresses

Reply:
We can, right !!